Jm update docs for net standard

masojus opened the pull request.

masojus committed:
Add a build status badge for Travis CI next to the one for AppVeyor.

masojus committed:
Remove references to the old PCL, netfx 4.5 and netfx 3.5 projects, and fix places where examples used the old namespace structure.

masojus committed:
Add some blurbs about using the SDK and contributing with the .NET Core SDK tools.

baumatron commented on CONTRIBUTING.md:
````diff
@@ -29,11 +29,20 @@ If a maintainer asks you to "rebase" your PR, they're saying that a lot of code
+Now, launch Visual Studio and open `Keen.sln`. Build the solution, discover unit tests, and run them. Or, if using the .NET Core SDK tools, build and make sure tests pass:
+
+```ssh
+$ dotnet build
````
Does this actually work these days from the root of the project?

masojus replied:
Yeah it will, but I should probably mention `error MSB3644` or explicitly passing `-f netstandard2.0` on platforms where you don't/can't have the right SDK/targeting pack for netfx4.5.

masojus replied:
Fun fact: running `dotnet test` from the root of the solution works too, which I didn't realize.

masojus committed:
A few minor clarifications:

- Call out an error/warning that occurs building the 4.5 target on non-Windows platforms.
- I noticed the NewSemVer script is trying to clean the `Debug` config instead of  `Release`.
- There were some lingering references to the old `netfx` config in the .csproj files.
